app/src/app/components/Toolbar/RecentMapsModal.tsx

```diff
@@ -174,22 +167,62 @@ const RecentMapsRow: React.FC<{
       <Table.Cell>
         <Text>{formattedDate}</Text>
       </Table.Cell>
-      <Table.Cell py=".5rem">
-        {!active && (
+      <Table.Cell py=".5rem" justify="center">
+        {!active ? (
           <Button
             onClick={() => onSelect(data)}
             variant="outline"
             className="box-content size-full rounded-xl hover:bg-blue-200 inline-flex transition-colors"
           >
-            Load
+            Ready to Load
           </Button>
+        ) : (
+          <div style={{}}>
+            <Button
+              disabled
+              variant="outline"
```

Comment:
Medium: ghost instead of outline to indicate no interaction
```suggestion
              variant="ghost"
```